content/en/blog/_posts/2024-09-20-kubeadm-v1beta4.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+title: 'Kubernetes v.31: kubeadm v1beta4'
+date: 2024-09-20
+slug: kubernetes-1-31-kubeadm-v1beta4
+author: >
+   Paco Xu (DaoCloud),
+---
+
+The [`kubeadm`](/docs/reference/setup-tools/kubeadm/) configuration
+([v1beta4](/docs/reference/config-api/kubeadm-config.v1beta4/))
+is released with Kubernetes v1.31. While v1beta3 is still supported,
+it will be deprecated and removed in future releases.
```

Comment:
```suggestion
but is marked as deprecated. It will be removed after a minimum
of 3 Kubernetes releases.
```

let's be very clear about this point.